esp-wifi/src/ble/controller/mod.rs

```diff
@@ -204,7 +204,13 @@ pub mod asynch {
                         }
                     }
                 }
-                warn!("[hci] error parsing packet: {:?}", e);
+                cfg_if::cfg_if! {
```

Comment:
I thought bt-hci speaks defmt, at least the main branch's current state does.

Reply:
mhhh - yeah probably we should activate that feature then 👍